app/modules/user/login/login.controller.js

```diff
@@ -19,8 +19,22 @@ const loginUser = async(req, res) => {
             } else if (!isMatch) {
                 res.send("Invalid login!")
 
+                res.send(403).json({
+                    success: false,
+                    message: 'Incorrect username or password'
+                });
+
 
             } else {
+
+                let token = json.sign({ username }, config.secret, {
```

Comment:
All the JWT related details should be added in the auth helper and access from that file. 
Also, create a response.js file and add all types of responses to that file and access from all the assigned controllers.

Also use http-status-codes npm package and access the response status from that rather than harcoding as 400, 500 etc.